src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,95 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_LOGIN = "marshmallow";
+    private static final Integer DEFAULT_AGE = 19;
+    private static final String DEFAULT_PASSWORD = "morning1";
+    private static final String UNACCEPTABLE_LOGIN = "bobby";
+    private static final Integer UNACCEPTABLE_AGE = 5;
+    private static final String UNACCEPTABLE_PASSWORD = "morn";
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.PEOPLE.clear();
+    }
+
+    @Test
+    void register_RegisterSameUserTwice_NotOk() {
+        User user = new User(DEFAULT_LOGIN, DEFAULT_PASSWORD, DEFAULT_AGE);
+        registrationService.register(user);
```

Comment:
you can use register() for testing data preparation. You should add the user to storage directly